### DISCUSSION TITLE
New maintainer/release manager: Marc MAZAS

### DISCUSSION THREAD

**Discussion by zosrothko** [javacc/javacc#268] [Announcements]
Hi everybody

Marc MAZAS @MarcMazas , a long time user and contributor to the JavaCC ecosystem has accepted to take in charge the release process and distribution to Maven Central of all artifacts of JavaCC7 and JavaCC8. 

Welcome Marc for taking this challenging duty.

**Comment by zbynek** [comment]
@MarcMazas thanks for taking care of this. How long do you want to continue the 7.0.x version and when do you plan to make 8.0 the main one? I'd suggest to do the switch soon to avoid having to make changes twice.

**Comment by MarcMazas** [comment]
Well, I'd like to do some laundry (e.g. scripts) and improvements (full java 8, better tests suite) on v7, port everything on v8 (e.g. C++ fixes), then make the v8 the main version. But I'm out of time for all this for a while :-(
